fem/complex_fem.hpp

```diff
@@ -241,13 +275,19 @@ class SesquilinearForm
                          OperatorHandle &A, Vector &X, Vector &B,
                          int copy_interior = 0);
 
+   void FormSystemMatrix(const Array<int> &ess_tdof_list,
+                         OperatorHandle &A);
+
    /** Call this method after solving a linear system constructed using the
        FormLinearSystem method to recover the solution as a ParGridFunction-size
        vector in x. Use the same arguments as in the FormLinearSystem call. */
    virtual void RecoverFEMSolution(const Vector &X, const Vector &b, Vector &x);
 
    virtual void Update(FiniteElementSpace *nfes = NULL);
 
+   /// Sets diagonal policy used upon construction of the linear system
+   void SetDiagonalPolicy(mfem::Matrix::DiagonalPolicy dpolicy);
```

Comment:
Should add a `GetDiagonalPolicy` method as well (just to be pedantic).

Reply:
Done